samza-core/src/main/java/org/apache/samza/container/LocalityManager.java

```diff
@@ -128,10 +125,9 @@ public void writeContainerToHostMapping(String containerId, String hostName) {
 
   public void close() {
     metadataStore.close();
-    taskAssignmentManager.close();
   }
 
-  public TaskAssignmentManager getTaskAssignmentManager() {
-    return taskAssignmentManager;
+  public Config getConfig() {
```

Comment:
Locality manager shouldn't expose config that was passed to it.

Reply:
Done.